04_verify.sh

```diff
@@ -186,11 +186,22 @@ EXPTD_V1ALPHAX_V1BETAX_CRDS="clusters.cluster.x-k8s.io \
   machines.cluster.x-k8s.io \
   machinesets.cluster.x-k8s.io \
   baremetalhosts.metal3.io"
-EXPTD_DEPLOYMENTS="capm3-system:capm3-controller-manager \
-  capi-system:capi-controller-manager \
-  capi-kubeadm-bootstrap-system:capi-kubeadm-bootstrap-controller-manager \
-  capi-kubeadm-control-plane-system:capi-kubeadm-control-plane-controller-manager \
-  baremetal-operator-system:baremetal-operator-controller-manager"
+# Add check for ironic deployment for Centos test
+# Different tests were failing in CI because of ironic deployment was not in ready state.
+if [[ "${EPHEMERAL_CLUSTER}" == "minikube" ]]; then
```

Comment:
Let's drop the if condition on line 257; it's not needed anymore.
here:
https://github.com/metal3-io/metal3-dev-env/blob/583c44b99029c633eb10adea6da848083231ce4c/04_verify.sh#L246

Reply:
I have discuss about this removal with Kashif and do the removal separately, as this is related to the separate test case. But I will do it in followups